pallets/nfts/src/features/transfer.rs

```diff
@@ -99,6 +108,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		// Update item details.
 		Item::<T, I>::insert(collection, item, &details);
+		Collection::<T, I>::insert(collection, &collection_details);
```

Comment:
Seems unnecessary as this function doesnt change anything on the collection details?